ga/18.0.0.4/oidcProvider/Dockerfile

```diff
@@ -12,21 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM websphere-liberty:18.0.0.4-kernel
+FROM websphere-liberty:kernel
```

Comment:
this should be `18.0.0.4-kernel` otherwise once we update `kernel` to 19.0.0.1, 19.0.0.2, etc, this would keep changing.

Reply:
fixed, thanks